### ISSUE TITLE
[a11y-defect-3]: Icon & link text is misleading or inappropriately used

### ISSUE THREAD

**Issue by ohgreatkate** [department-of-veterans-affairs/va.gov-team#64185]
### Point of contact

Kate Murphy @ohgreatkate

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

...
<img width="451" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/77993286/2fdf45eb-6cd4-4e11-bfd8-66bf652a6cab">

- Currently, the icon in use should be used for 'navigate page down' not 'download' 
- The text reads 'download VA Form 28-1900' but when the Veteran clicks the link they are redirected to another page to download the form. This is misleading.

### Reproduction steps

1. Go to the [introduction page of the form 28-1900](https://www.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/introduction)
2. click link [Apply for Veteran Readiness and Employment with VA Form 28-1900](https://www.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/) if directed to wizard
3. Scroll down to 'To apply by mail' section & review link
...

### Proposed solution or next steps

...
### Option 1: Recommended

- Remove icon
- Change link language to: 'Get VA Form 28-1900 to download'

### Option 2:
- Change icon to 'download' icon
- Change interaction to download the form directly on this page -- the reason this is not recommended is because it creates an opportunity for it to be missed when updating the paper form having it directly downloadable in multiple places.

### References, articles, or WCAG support

1. [WCAG SC 3.2.4 AA](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=246#consistent-identification)
...

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [X] Color, typography, & visual elements
- [X] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [X] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [X] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
